src/cirrus/docs/10_intro.rst

```diff
@@ -83,34 +86,56 @@ Relationship with stac-server
 
 Cirrus Workflows create STAC items, which are stored in S3 for persistence and
 can be published to `stac-server`_ (or any other STAC API) for indexing/search.
-In other words, Cirrus generates the data, stac-server makes it easily
-accessible to end-users and the whole world of STAC tooling.
+In other words, Cirrus generates the data, stac-server makes it
+accessible to end-users through the robust ecosystem of STAC tooling.
 
 .. _stac-server: https://github.com/stac-utils/stac-server
 
 
-Example use cases
------------------
+Example Earth Search use case
+----------------
+
+One prominent use case of Cirrus is as the processing pipeline for `Earth Search`_.
```

Comment:
Is it worth digging slightly deeper into how these map to cirrus concepts? maybe even a diagram of the specific Landsat workflow with USGS's SNS topic? (this can be a separate issue / improvement later)

Reply:
I think it would be -- I think that's one of the overall problems with these docs -- they do a good job of describing everything, but I don't they give a good picture of the dynamics of the system and how an individual pipeline would be implement and flow end-to-end. I think it's later work.